src/core/load-sprites.js

```diff
@@ -1,9 +1,15 @@
 export default (spritesUrl) => {
+  if (document.querySelector(".ably-sprites")) return;
+
   fetch(spritesUrl)
     .then((response) => response.text())
     .then((image) => {
       const container = document.createElement("div");
-      container.style.display = "none";
+      container.className = "ably-sprites";
+      container.style.opacity = 0;
```

Comment:
This was another requirement element of the spritesheet fix. Using `display: none` messes with the usage of SVGs within spritesheets, we need something less heavy-handed that's still invisible.